src/commands/dev/server_config/mod.rs

```diff
@@ -4,28 +4,36 @@ mod listening_address;
 use host::Host;
 use listening_address::ListeningAddress;
 
+use http::Uri;
+
 #[derive(Debug, Clone)]
 pub struct ServerConfig {
     pub host: Host,
     pub listening_address: ListeningAddress,
+    pub allowed_origins: Vec<Uri>,
 }
 
 impl ServerConfig {
     pub fn new(
         host: Option<&str>,
         ip: Option<&str>,
         port: Option<&str>,
+        allowed_origins: &[&str],
```

Comment:
starting to wonder if we want to have some sort of definition in `wrangler.toml` for stuff like this, seems like stuff you'd want to configure and share across a team

Reply:
having a [dev] block in the wrangler toml makes sense for stuff like this